addons/project/models/project.py

```diff
@@ -220,20 +222,15 @@ def _get_default_rating_status(self):
 
     # rating fields
     rating_request_deadline = fields.Datetime(compute='_compute_rating_request_deadline', store=True)
-    rating_status = fields.Selection([
-        ('stage', 'Rating when changing stage'),
-        ('periodic', 'Periodical Rating'),
-        ('no', 'No rating')], string='Customer Ratings', default=_get_default_rating_status, required=True,
-        help="How to get customer feedback?\n"
-                 "- Rating when changing stage: an email will be sent when a task is pulled in another stage.\n"
-                 "- Periodical Rating: email will be sent periodically.\n\n"
-                 "Don't forget to set up the mail templates on the stages for which you want to get the customer's feedbacks.")
+    rating_active = fields.Boolean('Customer Ratings', default=True)
+    rating_status = fields.Selection([('stage', 'Rating when changing stage'), ('periodic', 'Periodical Rating')], 'Customer Ratings Status', help="How to get customer feedback?\n"
```

Comment:
You need to write a migration script here. What do we do with all the existing projects with a rating status set to ``'no'``. 

My guess is that it should be set to ``NULL`` instead, with ``rating_active`` set to ``False``, but we have to do it :)

Reply:
Done. The default is 'stage'.